src/FileUtils/AgGridUtils.js

```diff
@@ -6,20 +6,36 @@ import XLSXUtils from './XLSXUtils';
 import { ValidationUtils } from '../ValidationUtils';
 import { Error as PanelError } from '../models';
 
-const _buildColumnsCountError = (missingColsCount, rowIndex, expectedColsCount, colsCount, expectedColsName, row) => {
-  const errorSummary = `Missing field${missingColsCount > 1 ? 's' : ''}`;
+const _buildEmptyFieldError = (rowIndex, expectedColsName, colIndex) => {
+  const errorSummary = `Empty field`;
+  const errorLoc = `Line ${rowIndex}, Column ${colIndex + 1} ("${expectedColsName.split(',')[colIndex]}")`;
```

Comment:
Calling `expectedColsName.split(',')` will cause bugs if a column name contains a comma character `,`.
Instead of `split`-ting the `expectedColsName` **string** here, it would be more robust to add an additional argument to this function to receive the **array** of columns names (it can be found in `_forgeColumnsCountError` by doing `expectedCols.map((col) => col.field)`).